domain/BUILD.bazel

```diff
@@ -0,0 +1,239 @@
+    src = [
+        "src/test/java/org/oppia/domain/classify/rules/dragAndDropSortInput/"
+        + "DragDropSortInputHasElementXBeforeElementYRuleClassifierProviderTest.kt"
+    ],
+    test_class = "org.oppia.domain.classify.rules.dragAndDropSortInput."
+    + "DragDropSortInputHasElementXBeforeElementYRuleClassifierProviderTest",
+)
+
+domain_test(
+    name = "DragDropSortInputIsEqualToOrderingClassifierProviderTest",
+    src = [
+        "src/test/java/org/oppia/domain/classify/rules/dragAndDropSortInput/"
+        + "DragDropSortInputIsEqualToOrderingClassifierProviderTest.kt"
+    ],
+    test_class = "org.oppia.domain.classify.rules.dragAndDropSortInput."
+    + "DragDropSortInputIsEqualToOrderingClassifierProviderTest",
+)
+
+domain_test(
+    name = "DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTest",
+    src = [
+        "src/test/java/org/oppia/domain/classify/rules/dragAndDropSortInput/"
+        + "DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTest.kt"
+    ],
+    test_class = "org.oppia.domain.classify.rules.dragAndDropSortInput."
+    + "DragDropSortInputIsEqualToOrderingWithOneItemIncorrectClassifierProviderTest",
+)
+
+domain_test(
+    name = "ImageClickInputIsInRegionRuleClassifierProviderTest",
+    src = [
+        "src/test/java/org/oppia/domain/classify/rules/imageClickInput/"
+        + "ImageClickInputIsInRegionRuleClassifierProviderTest.kt"
+    ],
+    test_class = "org.oppia.domain.classify.rules.imageClickInput."
+    + "ImageClickInputIsInRegionRuleClassifierProviderTest",
+)
+
+domain_test(
+    name = "MultipleChoiceInputEqualsRuleClassifierProviderTest",
+    src = [
+        "src/test/java/org/oppia/domain/classify/rules/multiplechoiceinput/"
+        + "MultipleChoiceInputEqualsRuleClassifierProviderTest.kt"
+    ],
+    test_class = "org.oppia.domain.classify.rules.multiplechoiceinput."
+    + "MultipleChoiceInputEqualsRuleClassifierProviderTest",
+)
+
+domain_test(
+    name = "TextInputContainsRuleClassifierProviderTest",
+    src = [
+        "src/test/java/org/oppia/domain/classify/rules/textinput/"
+        + "TextInputContainsRuleClassifierProviderTest.kt"
+    ],
+    test_class = "org.oppia.domain.classify.rules.textinput."
+    + "TextInputContainsRuleClassifierProviderTest",
+)
+
+domain_test(
+    name = "TextInputEqualsRuleClassifierProviderTest",
+    src = [
+        "src/test/java/org/oppia/domain/classify/rules/textinput/"
+        + "TextInputEqualsRuleClassifierProviderTest.kt"
+    ],
+    test_class = "org.oppia.domain.classify.rules.textinput."
+    + "TextInputEqualsRuleClassifierProviderTest",
+)
+
+#TODO: Building not passing
+domain_test(
+    name = "ExplorationDataControllerTest",
+    src = ["src/test/java/org/oppia/domain/exploration/ExplorationDataControllerTest.kt"],
+    test_class = "org.oppia.domain.exploration.ExplorationDataControllerTest",
+)
+
+#TODO: Building not passing
```

Comment:
Can we have a TODO number mention here to keep track of things?

Reply:
I believe these tests are actually passing & I forgot to remove the TODOs! So Sorry! Fixing this now